app/views/articles/show.html.erb

```diff
@@ -9,9 +9,4 @@
 </div>
 <div class="govuk-body">
   <%= translate_markdown(@article.markdown) %>
-</div>
-<div class="govuk-grid-row govuk-!-padding-top-6 govuk-!-margin-bottom-8">
-  <div class="govuk-grid-column-full">
-    <%== insert_block('other_useful_resources') %>
-  </div>
-</div>
+</div>
```

Comment:
There should be an empty line at the end of the file. Some xNIX system get upset by a lack of trailing empty line, so usually best practice to have one.

Reply:
Good spot. Just resolved it